### ISSUE TITLE
One more segfault =)

### ISSUE THREAD

**Issue by eerimoq** [qchateau/cbitstruct#10]
Found a segfault in version 1.0.2:

``` python
cbitstruct.byteswap('\x02\x02', b'z')
```
I'm using [pyfuzzer](https://github.com/eerimoq/pyfuzzer), a fuzz tester I recently implemented, to find all these corner cases. It's actually quite useful =)

**Comment by qchateau** [comment]
Damn you're giving me work x) nice tool though :)

**Comment by qchateau** [comment]
I tried to toy with pyfuzzer, it complained a lot about integer overflow being UB, even though it is not relevant. Is there any way to disable it ?

```
$ pyfuzzer run -l max_total_time=120  _cbitstruct _cbitstruct.c
...
#66848	REDUCE cov: 219 ft: 475 corp: 148/1369b lim: 29 exec/s: 66848 rss: 52Mb L: 14/26 MS: 1 EraseBytes-
#67057	REDUCE cov: 219 ft: 475 corp: 148/1368b lim: 29 exec/s: 67057 rss: 52Mb L: 10/26 MS: 4 ShuffleBytes-InsertByte-ChangeASCIIInt-EraseBytes-
#67191	REDUCE cov: 219 ft: 475 corp: 148/1366b lim: 29 exec/s: 67191 rss: 52Mb L: 12/26 MS: 4 ChangeBit-ChangeByte-ChangeBit-EraseBytes-
	NEW_FUNC[1/1]: 0x46bf10  (/home/quentin/dev/cbitstruct/cbitstruct/pyfuzzer+0x46bf10)
#69941	NEW    cov: 226 ft: 482 corp: 149/1393b lim: 29 exec/s: 69941 rss: 54Mb L: 27/27 MS: 5 InsertRepeatedBytes-CrossOver-ChangeByte-ChangeBinInt-ChangeBinInt-
#71042	REDUCE cov: 226 ft: 482 corp: 149/1391b lim: 29 exec/s: 71042 rss: 54Mb L: 25/26 MS: 1 CrossOver-
_cbitstruct.c:55:19: runtime error: signed integer overflow: 777777777 * 10 cannot be represented in type 'int'
MS: 2 CopyPart-InsertRepeatedBytes-; base unit: c6d866071de91876204118978e7fbe65575a8f5c
0xe,0x3,0x1,0xd4,0x60,0x70,0x70,0x70,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x37,0x70,0x70,0x70,0x70,0x70,
\x0e\x03\x01\xd4`ppp7777777777777777ppppp
artifact_prefix='./'; Test unit written to ./crash-271e4942667c12aae87fa4705dc0b714b9f9796a
Base64: DgMB1GBwcHA3Nzc3Nzc3Nzc3Nzc3Nzc3cHBwcHA=
error: Command '['./pyfuzzer', 'corpus', '-ignore_crashes', '-max_total_time=120']' returned non-zero exit status 1.
```

**Comment by eerimoq** [comment]
Comment this line and it will not test for integer overflows:

https://github.com/eerimoq/pyfuzzer/blob/master/pyfuzzer/__init__.py#L66

**Comment by qchateau** [comment]
Maybe you could find some way to let the user change these flags without hacking the code. 

I fixed the segfault in 1.0.3 btw
